examples/notebooks/beam-ml/custom_remote_inference.ipynb

```diff
@@ -308,11 +305,10 @@
       "source": [
         "### Manage batching\n",
         "\n",
-        "Before we can chain together the pipeline steps, we need to understand batching.\n",
-        "When running inference with your model, either in Apache Beam or in an external API, you can batch your input to increase the efficiency of the model execution.\n",
-        "`RunInference` PTransform manages batching in this pipeline with `BatchElements` transform to group elements together and form a batch of the desired size.\n",
+        "When you run inference with your model, either in Apache Beam or in an external API, batch your input to increase the efficiency of the model execution.\n",
+        "In this example, the `RunInference PTransform` manages batching by using the `BatchElements` transform to group elements together and form a batch of the desired size.\n",
```

Comment:
I think this might be a little less clear than we could be. RunInference will always handle batching for us (not just in this example), and users should never need to take any action to get this batching. I think this reads more as "users sometimes need to batch inputs, but in this case RunInference handles it".

Maybe we could just change this sentence to something like:

```
The `RunInference PTransform` automatically manages batching by using the `BatchElements` transform to dynamically group elements together into batches based on the throughput of the pipeline.
```

Note I also added a bit about dynamic batching because I think its a cool selling point of RunInference that folks might find useful. Feel free to word however you see fit.

Reply:
Updated